src/Raven.Server/Documents/Handlers/TransactionsRecordingHandler.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Threading.Tasks;
+using Raven.Server.Routing;
+using Raven.Server.ServerWide.Context;
+
+namespace Raven.Server.Documents.Handlers
+{
+    public class TransactionsRecordingHandler : DatabaseRequestHandler
+    {
+        [RavenAction("/databases/*/transactions/replay", "POST", AuthorizationStatus.ValidUser)]
+        public Task ReplayRecording()
+        {
+            var replayStream = RequestBodyStream();
+            //Todo to use zip
+            //using (var gZipStreamDocuments = new GZipStream(fileStream, CompressionMode.Compress, true))
+            Database.TxMerger.Replay(replayStream);
```

Comment:
This need to be an operation, with status reporting going on

Reply:
In progress